modules/executors/README.md

```diff
@@ -0,0 +1,12 @@
+# Executors module
+
+This module provides the resources to provision [Sourcegraph executor](https://docs.sourcegraph.com/admin/executors) compute resources on Google Cloud. For a high-level overview of the resources deployed by this module, see the [root module](https://registry.terraform.io/modules/sourcegraph/executors/google/latest). This module includes the following resources:
+
+- Google compute instance template
+- Google compute group manager, and auto-scaler
+- Google compute firewall
+- Google service account membership to enable log and metric writes
+
+This module does **not** automatically create networking or Docker mirror resources. The `network_id`, `subnet_id`, and `docker_registry_mirror` variables must be supplied explicitly with resources that have been previously created.
+
+This module is often used with the sibling modules that create [networking](https://registry.terraform.io/modules/sourcegraph/executors/google/latest/submodules/networking) and [Docker mirror](https://registry.terraform.io/modules/sourcegraph/executors/google/latest/submodules/docker-mirror) resources which can be shared by multiple instances of the executor module (listening to different queues or being deployed in a different environment).
```

Comment:
Want to link to `#inputs` instead of the README? Or maybe add text to the README? The page currently looks like this:

<img width="687" alt="CleanShot 2021-11-23 at 11 47 38@2x" src="https://user-images.githubusercontent.com/1387653/143085492-16be77ec-c656-49a8-b03b-c06fd4a2ac19.png">

Reply:
Already tracked in https://github.com/sourcegraph/sourcegraph/issues/27440